taf/cli.py

```diff
@@ -0,0 +1,21 @@
+import click
+from taf.repository_tool import load_repository
+
+
+@click.group()
+def cli():
+  pass
+
+
+@cli.command()
+@click.option('--repo-path', help='Authentication repository\'s path')
+@click.option('--file-path', help='Target file\'s path, relative to the targets directory')
+@click.option('--keystore-path', help='Path of the keystore file')
+@click.option('--targets-role', default='targets', help='Targets metadata file to be updated')
+def add_target_file(repo_path, file_path, keystore_path, targets_role):
```

Comment:
Should we make first three click options required here? Or we can use `click.argument` for those.

Reply:
The first three parameters are required. Yes, I think it makes sense to use `argument`